recipes/systemd/all/conanfile.py

```diff
@@ -0,0 +1,57 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+
+
+class SystemdConan(ConanFile):
+    name = "systemd"
+    version = "system"
+    description = "System and service manager"
+    topics = ("systemd", "udev")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://systemd.io/"
+    license = "LGPL-2.1-or-later"
+    settings = {"os": "Linux"}
+
+    def system_requirements(self):
+        if tools.os_info.is_linux and self.settings.os == "Linux":
+            package_tool = tools.SystemPackageTool(conanfile=self, default_mode="verify")
+            if tools.os_info.with_yum or tools.os_info.with_dnf:
+                packages = ["systemd", "systemd-libs", "libudev-devel"]
+            elif tools.os_info.with_apt:
+                packages = ["libsystemd-dev", "libudev-dev"]
+            elif tools.os_info.with_pacman:
+                packages = ["systemd", "systemd-libs"]
+            elif tools.os_info.with_zypper:
+                packages = ["systemd-devel", "libudev-devel"]
```

Comment:
```suggestion
                packages = ["systemd-devel", "libudev-devel", "systemd"]
```
systemd package provides the .pc file https://opensuse.pkgs.org/15.2/opensuse-oss-x86_64/systemd-234-lp152.30.1.x86_64.rpm.html